apps/events/models.py

```diff
@@ -0,0 +1,19 @@
+from ckeditor.fields import RichTextField
+from django.contrib.auth.models import Group
+from django.db import models
+
+
+class Event(models.Model):
+    name = models.CharField(max_length=255)
+    image = models.ImageField(upload_to="events")
+    description = RichTextField()
+    lead = models.TextField()
+    start_time = models.DateTimeField()
+    end_time = models.DateTimeField()
+    member_only = models.BooleanField()
+    location = models.CharField(max_length=255)
+    organizer = models.ForeignKey(Group, blank=True, null=True, on_delete=models.SET_NULL)
```

Comment:
Should this field be chosen by the user, or set by default according to what group the user belongs in? As in the latter case we need to set it in the save function of the model.

Reply:
I think it's fine if we leave the user choose the organizer field by themselves. Wouldn't make much of a difference, and anybody with permission to make events is trustworthy